### PR TITLE
Converted item_combo_db txt to yml

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/1922

* **Server Mode**: both

* **Description of Pull Request**: 

Continuation of https://github.com/rathena/rathena/pull/5992 on a new branch.

  * Converts item_combo_db.txt into YAML.
  * Includes CSV2YAML converter.
 
--------------------------------------------------------------------------------------
TODO
 - [x] Remove txt files before merge

### CODE REVIEW

File: src/map/itemdb.cpp [hunk 1]
```diff
@@ -1932,142 +1917,164 @@ static bool itemdb_read_noequip(char* str[], int columns, int current) {
...
-	sprintf(path, "%s/%s", basedir, "item_combo_db.txt");
+	if (items.size() < 2) {
+		this->invalidWarning(node[nodeName], "Not enough item to make a combo (need at least 2). Skipping.\n");
+		return false;
+	}
 
-	if ((fp = fopen(path, "r")) == NULL) {
-		if(silent==0) ShowError("itemdb_read_combos: File not found \"%s\".\n", path);
-		return;
+	if (items.size() > MAX_ITEMS_PER_COMBO) {
```
Comment (aleos89) [inline src/map/itemdb.cpp:1955]: Do we need this still? Since it’s a vector we shouldn’t be concerned of a max limit anymore.

**Comment by Atemo** [comment]
Thumbs up if you prefer
```
  - Combo:
      - Inverse_Scale
      - Helm_Of_Dragoon
    Script: >-
      bonus bAspd,1;
      bonus bFlee2,3;

  - Combo:
      - Legacy_Of_Dragon
      - Helm_Of_Dragoon
    Script: >-
      bonus bAspd,1;
      bonus bFlee2,3;

  - Combo:
      - Dragon_Wing
      - Helm_Of_Dragoon
    Script: >-
      bonus bAspd,1;
      bonus bFlee2,3;
 ```
Thumbs down if you prefer 
```
  - Combos:
      - Combo:
          - Inverse_Scale
          - Helm_Of_Dragoon
      - Combo:
          - Legacy_Of_Dragon
          - Helm_Of_Dragoon
      - Combo:
          - Dragon_Wing
          - Helm_Of_Dragoon
    Script: >-
      bonus bAspd,1;
      bonus bFlee2,3;
```
Any comment about the format is welcome

File: src/map/itemdb.cpp (hunk 1 above)
Comment (aleos89) [reply src/map/itemdb.cpp:1955]: Yeah, I say remove it, if we can just go back and verify we don't have any other limiters, why cap it? :)

**Comment by Everade** [comment]
To the voting i would like to add:
It would be cumbersome if certain combos would be split up because of (thumbs down).

I mean, not that we make ASPD: 1 and then include every single combo in that list, even though those specific combos include more bonuses. That would result in a huge scatter of information rather than keeping it all in one spot.

**Review by Lemongrass3110 — APPROVED** [review]

**Comment by Lemongrass3110** [comment]
seal

**Review by aleos89 — APPROVED** [review]

**Pull request merged**